kubeadm/flannel/flannel-overlay.yml

```diff
@@ -63,6 +82,40 @@ data:
         ]
       }
     }
+  cni-conf-containerd.json: |
+    {
+      "name": "flannel.4096",
+      "cniVersion": "0.2.0",
+      "type": "flannel",
+      "capabilities": {
+        "portMappings": true,
+        "dns": true
+      },
+      "delegate": {
+        "type": "sdnoverlay",
+        "AdditionalArgs": [
+          {
+            "Name": "EndpointPolicy",
+            "Value": {
+              "Type": "OutBoundNAT",
+              "Settings" : {
+                "Exceptions": []
+              }
+            }
+          },
+          {
+            "Name": "EndpointPolicy",
+            "Value": {
+              "Type": "SDNROUTE",
+              "Settings": {
+                "DestinationPrefix": "",
+                "NeedEncap": true
+              }
+            }
+          }
```

Comment:
Should there be PA address specified here as well?

See https://github.com/microsoft/windows-container-networking/blob/master/example/flannel_overlay.conf

Reply:
I can add it if you think it would be of benefit.